QSB/PlayerRegistry.cs

```diff
@@ -26,21 +19,22 @@ public static void RegisterPlayerBody(uint id, GameObject body)
 
         public static bool PlayerExists(uint id)
         {
-            return playerList.Any(x => x.NetId == id);
+            return PlayerList.Any(x => x.NetId == id);
         }
 
         public static void CreatePlayer(uint id, string name)
         {
-            if (!PlayerExists(id))
+            if (PlayerExists(id))
             {
-                DebugLog.ToConsole($"Creating player: {id}");
-                var player = new PlayerInfo()
-                {
-                    NetId = id,
-                    Name = name
-                };
-                playerList.Add(player);
+                return;
             }
+            DebugLog.ToConsole($"Creating player: {id}");
+            var player = new PlayerInfo()
```

Comment:
In GameState you removed the (), but here you've left them... which is right?

Reply:
without :)